client/src/js/services/grid/TransactionService.js

```diff
@@ -207,6 +207,26 @@ function TransactionService($timeout, util, uiGridConstants, bhConstants, Notify
   }
 
   /**
+   * @function editCellGrid
+   *
+   * @description
+   * Edit cell value for custom template
+   *
+   * @public
+   *
+   */
+  Transactions.prototype.editCellGrid = function editCellGrid(rowEntity, field, value) {
```

Comment:
Can you make this function the same function as `editCell`?  It looks duplicated.  And if it must be a different function, can this just call `editCell` internally?